### DISCUSSION TITLE
Get just one skin by their ID

### DISCUSSION THREAD

**Discussion by ClaudioRibeiro2** [ByMykel/CSGO-API#118] [Q&A] [answered]
Hello, there is a way to get just one skin with the API call?
Ex: bymykel.github.io/CSGO-API/api/pt-BR/skin-65604

**Comment by ByMykel** [comment, marked as answer]
No, unfortunately is not posible as is just a static JSON file.

**Reply by ClaudioRibeiro2** [reply]
Okay, apreciated for your fast response.
You API is saving me for my college projetc alredy

**Reply by ByMykel** [reply]
Are you building also a backend for your project?

You can use just this API but some files are very large making the requests slow. Check this [example](https://bymykel.github.io/counter-strike-items/#/).

**Reply by ClaudioRibeiro2** [reply]
Hey, very cool your project, i gonna use as inspiration for my college project.
Thank you for your support!
